[dper3] Create dper LearningRate low-level module

esqu1 opened the pull request:
Summary: As title; this will unblock migration of several modules that need learning rate functionality.

Test Plan:
```
buck test //dper3/dper3/modules/low_level_modules/tests:learning_rate_test
```

Differential Revision: D23681733

houseroad commented on caffe2/sgd/learning_rate_op.cc:
```diff
@@ -163,8 +163,10 @@ C10_EXPORT_CAFFE2_OP_TO_C10_CPU(
+    "float? decay = 1.0, "
     "float? start_multiplier = 0, "
     "float? end_mulitplier = 0, "
+    "float? end_multiplier = 0, "
```
wait, why we double end_multiplier?

esqu1 replied:
I had this because I thought that deleting the original typo'ed end_multiplier would break backwards compatibility; is it safe to just rename, or is there something else that has to be done?

houseroad commented on caffe2/sgd/learning_rate_op.cc:
```diff
@@ -163,8 +163,10 @@ C10_EXPORT_CAFFE2_OP_TO_C10_CPU(
     "int? inactive_period = -1, "
     "int? max_iter = -1, "
     "int? num_iter = 0, "
+    "float? decay = 1.0, "
```
please add new parameter at the end, and make sure the default value matches the original behavior.

houseroad reviewed (commented):
Current change is indeed BC breaking.

Commit:
[dper3] Create dper LearningRate low-level module (#44639)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/44639

As title; this will unblock migration of several modules that need learning rate functionality.

Test Plan:
```
buck test //dper3/dper3/modules/low_level_modules/tests:learning_rate_test
```

Differential Revision: D23681733

fbshipit-source-id: 860058d744fbc6e1be9835fbe247d92b6a206cf0
